src/base-input.js

```diff
@@ -84,6 +85,8 @@ export default class BaseInput extends LitElement {
       label:                   { type: String },
       name:                    { type: String },
       type:                    { type: String },
+      format:                  { type: Object },
```

Comment:
These properties were grouped by Sting then Boolean. Wondering why these were places here?

Reply:
Moved to after list of strings